### ISSUE TITLE
update brute force docs to reflect available lockouts modes (temporary / permanent / mixed) 

### ISSUE THREAD

**Issue by gilvansfilho** [keycloak/keycloak#27378]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

docs

### Describe the bug

We should update brute force docs to reflect changes made by https://github.com/keycloak/keycloak/pull/26208
Docs show that image of brute force admin UI:

<img width="1179" alt="image" src="https://github.com/keycloak/keycloak/assets/6987164/c36ea2ad-9699-4d8e-a01f-5f81f77082ee">

But the UI has changed, as below:

<img width="1925" alt="image" src="https://github.com/keycloak/keycloak/assets/6987164/c67ae6d6-672b-4b03-ae91-c809202285d4">

<img width="1929" alt="image" src="https://github.com/keycloak/keycloak/assets/6987164/c1db7a2d-469b-4274-9f76-1e46e48019f7">

<img width="1930" alt="image" src="https://github.com/keycloak/keycloak/assets/6987164/f6d9432f-53da-45c8-8b8b-0036be0b1d0e">

<img width="1927" alt="image" src="https://github.com/keycloak/keycloak/assets/6987164/f8e7bce5-458a-433b-b5ee-89418020c6ad">

### Version

24.0.0

### Regression

- [ ] The issue is a regression

### Expected behavior

N/A

### Actual behavior

N/A

### How to Reproduce?

Build main branch and see brute force admin UI

### Anything else?

I could see that because I am working on #25917

**Comment by gilvansfilho** [comment]
@pedroigor @douglaspalmer FYI

**Comment by pedroigor** [comment]
@mposolda @douglaspalmer Please, let me know if you want me to work on this.

**Comment by andymunro** [comment]
@gilvansfilho Do you have time to follow up on this issue?  It's an important one to address.

**Comment by gilvansfilho** [comment]
Sure but waiting someone take a look on this first https://github.com/keycloak/keycloak/pull/31936

**Comment by rmartinc** [comment]
@gilvansfilho Are you going to work on this once https://github.com/keycloak/keycloak/pull/31936 is merged?

**Comment by gilvansfilho** [comment]
Sure. But this week I am a bit busy so I think next week I will be able to handle this.

**Comment by gilvansfilho** [comment]
@andymunro @rmartinc  PR was merged before I made the requested changes. Should I submit a new PR?

**Comment by ahus1** [comment]
@douglaspalmer / @rmartinc - this is marked for backporting to KC26. If this should be done, could one of you please pick up this task, or remove the backport label if this is no longer planned?

**Comment by gilvansfilho** [comment]
@ahus1 I think that when this https://github.com/keycloak/keycloak/pull/31936 was done KC 26 had already be released. Updated docs mention this. 

If so should not this be backported too?

**Comment by ahus1** [comment]
Looking at this in more details, this is about a feature, and we usually only backport bug fixes. So I'm removing the backporting label.
